replications/internal/queue_management.go

```diff
@@ -178,7 +178,7 @@ func (rq *replicationQueue) SendWrite() bool {
 
 	// Update metrics after the call to scan.Advance()
 	defer func() {
-		rq.metrics.Dequeue(rq.id, rq.queue.DiskUsage())
+		rq.metrics.Dequeue(rq.id, rq.queue.TotalBytes())
```

Comment:
`TotalBytes` seems better for this metric since it reflects how many bytes are left to process in the queue.